endpoints/makegoods.yaml

```diff
@@ -247,212 +247,71 @@ paths:
               schema:
                 $ref: 'schemas/commonSchemas.yaml#/components/schemas/Error'
 
-          
-  /v1/makegoods/subscribe:
+  /v1/makegoods/subscriptions:
       post:
-          summary: Subscribe to makegoods events
-          description: Subscribe to makegoods events
-          operationId: makegoodsSubscribe
+          summary: Create a new subscription
+          description: Create a new subscription
+          operationId: ordersSubscription
           requestBody:
-            description: makegoodsSubscribe
+            description: Create a new subscription
             required: true
             content:
               application/json:
                 schema:
-                  oneOf:
-                    - $ref: 'schemas/makegoodsSchemas.yaml#/components/schemas/makegoodsSubscribeSGS'
-                    - $ref: 'schemas/makegoodsSchemas.yaml#/components/schemas/makegoodsSubscribeSGSCounterParty'
+                  $ref: 'schemas/commonSchemas.yaml#/components/schemas/subscriptionRequest'
           responses:
             '202':
```

Comment:
if this is a synchronous request, we need to use `200`, not `202`

Reply:
should it be 201 as per description?